### PR TITLE
fixed ScannerError issue while running tests for cfme-5.7.1 and above

### PR DESCRIPTION
**Pull request by hroyrh**
This commit is to resolve the issue mentioned here : https://github.com/redhat-performance/cfme-performance/issues/4
@arcolife @psuriset can you merge it.

### CODE REVIEW

**Comment by arcolife** [comment]
@hroyrh ssh client can handle stderr. Please update the PR to do that instead. 
This is not a recommended workaround as the error string may change in future versions of CFME.

File: cfme-performance/utils/ssh.py [hunk 1]
```diff
@@ -128,7 +128,7 @@ def run_command(self, command, timeout=RUNCMD_TIMEOUT, reraise=False, log_less=F
 
                 if session.recv_stderr_ready:
                     for line in stderr:
-                        output += line
+                        #output += line
```
Comment (arcolife) [inline cfme-performance/utils/ssh.py]: or what I'd do is simply include a default option "ignore_stderr"=False here -> https://github.com/hroyrh/cfme-performance/blob/53de4c87f4dc7d587eef616a07b1f78562bafacd/cfme-performance/utils/ssh.py#L106

And then activate that just for yaml load. This way we'd be sure of this executing only as required.

File: cfme-performance/utils/ssh.py [hunk 2]
```diff
@@ -128,7 +128,10 @@ def run_command(self, command, timeout=RUNCMD_TIMEOUT, reraise=False, log_less=F
 
                 if session.recv_stderr_ready:
                     for line in stderr:
-                        #output += line
+                        if ignore_stderr:
+                            continue
```
Comment (arcolife) [inline cfme-performance/utils/ssh.py]: could you put this if condition before the for loop? there's no point in looping N times if the std err is True.

File: cfme-performance/utils/ssh.py [hunk 3]
```diff
@@ -152,7 +155,7 @@ def run_command(self, command, timeout=RUNCMD_TIMEOUT, reraise=False, log_less=F
     def run_rails_command(self, command, timeout=RUNCMD_TIMEOUT):
         logger.info("Running rails command `%s`", command)
         return self.run_command('cd /var/www/miq/vmdb; bin/rails runner {}'.format(command),
-            timeout=timeout)
+            timeout=timeout, ignore_stderr=True)
```
Comment (arcolife) [inline cfme-performance/utils/ssh.py]: adding a comment here specifying the reason for including this, would be helpful. Just a mentioned of the issue like this: 
```# https://github.com/redhat-performance/cfme-performance/issues/4 ```

Also, ignoring the error is only required for getting the config for yaml_data as per #4 
So running this for all rails command isn't fair. Don't you think? I would further channelize this to https://gist.github.com/hroyrh/56363764f0b1ec5ab058b6e59a37cef1#file-scannererror-L36 if possible.

Comment (hroyrh) [inline cfme-performance/utils/ssh.py]: I have made the changes, let me know if there is any other issue

**Comment by arcolife** [comment]
@hroyrh LGTM. just squash the commits into a single one before we merge it.

**Comment by hroyrh** [comment]
@arcolife i have squashed all the previous commits

**Pull request merged**